ElectronicObserver/Window/Dialog/ShipDataPicker/ShipDataPickerView.xaml

```diff
@@ -0,0 +1,160 @@
+		</Style>
+
+		<Style BasedOn="{StaticResource DefaultCheckBoxStyle}" TargetType="CheckBox">
+			<Setter Property="Margin" Value="4 0" />
+			<Setter Property="Padding" Value="2 0" />
+			<Setter Property="MinWidth" Value="0" />
+		</Style>
+	</Window.Resources>
+
+	<Grid>
+		<Grid.RowDefinitions>
+			<RowDefinition Height="Auto" />
+			<RowDefinition />
+		</Grid.RowDefinitions>
+
+		<filter:ShipFilterControl Grid.Row="0" DataContext="{Binding Filters}" />
+
+		<DataGrid
+			Grid.Row="1"
+			AutoGenerateColumns="False"
+			HeadersVisibility="Column"
+			IsReadOnly="True"
+			ItemsSource="{Binding ShipsFiltered}"
+			SelectedItem="{Binding SelectedShip}"
+			SelectionMode="Single"
+			>
+			<DataGrid.ColumnHeaderStyle>
+				<Style BasedOn="{StaticResource DefaultDataGridColumnHeaderStyle}" TargetType="{x:Type DataGridColumnHeader}">
+					<Setter Property="Foreground" Value="{Binding DataContext.FontBrush, Source={StaticResource Proxy}}" />
+					<Setter Property="FontFamily" Value="{Binding DataContext.Font, Source={StaticResource Proxy}}" />
+					<Setter Property="FontSize" Value="{Binding DataContext.FontSize, Source={StaticResource Proxy}}" />
+					<Setter Property="Padding" Value="5 0" />
+					<Setter Property="Template" Value="{StaticResource DataGridColumnHeaderTemplate}" />
+					<Style.Triggers>
+						<Trigger Property="SortDirection" Value="{x:Null}">
+							<Setter Property="Padding" Value="5 0" />
+						</Trigger>
+					</Style.Triggers>
+				</Style>
+			</DataGrid.ColumnHeaderStyle>
+
+			<ui:DataGridHelper.CheckBoxColumnElementStyle>
+				<Style BasedOn="{StaticResource DataGridReadOnlyCheckBoxStyle}" TargetType="CheckBox">
+					<Setter Property="Margin" Value="0" />
+					<Setter Property="Padding" Value="0" />
+					<Setter Property="HorizontalAlignment" Value="Center" />
+				</Style>
+			</ui:DataGridHelper.CheckBoxColumnElementStyle>
+
+			<ui:DataGridHelper.TextColumnElementStyle>
+				<Style BasedOn="{StaticResource DataGridTextBlockStyle}" TargetType="TextBlock">
+					<Setter Property="Padding" Value="5 0" />
+					<Setter Property="FontFamily" Value="{Binding DataContext.Font, Source={StaticResource Proxy}}" />
+					<Setter Property="FontSize" Value="{Binding DataContext.FontSize, Source={StaticResource Proxy}}" />
+					<Setter Property="Foreground" Value="{Binding DataContext.FontBrush, Source={StaticResource Proxy}}" />
+				</Style>
+			</ui:DataGridHelper.TextColumnElementStyle>
+
+			<DataGrid.InputBindings>
+				<MouseBinding Command="{Binding SelectShipCommand}" MouseAction="LeftDoubleClick" />
+			</DataGrid.InputBindings>
+
+			<DataGrid.Columns>
+				<DataGridTextColumn
+					Binding="{Binding Ship.ID}"
+					FontFamily="{Binding DataContext.Font, Source={StaticResource Proxy}}"
+					FontSize="{Binding DataContext.FontSize, Source={StaticResource Proxy}}"
+					Foreground="{Binding DataContext.FontBrush, Source={StaticResource Proxy}}"
```

Comment:
There's no need to set `FontFamily`, `FontSize` and `Foreground` for the columns.